CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
-## 0.8.0 (Unreleased)
+## 0.8.0 (2022-07-22)
 
 ### New Features:
```

Comment:
In the past, we have ended all sentences with a period. Let's do this for all of these bullet points too.